### PR TITLE
Telegram

### PR DESCRIPTION
**Pull request by tyouxa**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

notification/telegram

##### ANSIBLE VERSION

```
ansible 2.1.2.0
```

##### SUMMARY

Add new flags disable_web_page_preview and disable_notification to telegram module

```

```

### CODE REVIEW

File: notification/telegram.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 DOCUMENTATION = """
 
 module: telegram
-version_added: "2.2"
```
Comment (alxgu) [inline notification/telegram.py:26]: This version number shouldn't increased

File: notification/telegram.py [hunk 2]
```diff
@@ -45,6 +45,23 @@
     description:
       - Telegram group or user chat_id
     required: true
+  disable_web_page_preview:
```
Comment (alxgu) [inline notification/telegram.py:48]: Add here version_added: "2.3"

File: notification/telegram.py [hunk 3]
```diff
@@ -45,6 +45,23 @@
...
+    description:
+      - Disables link previews for links in this message
+    required: false
+    default: 'True'
+    choices:
+      - 'True'
+      - 'False'
+  disable_notification:
```
Comment (alxgu) [inline notification/telegram.py:57]: Add here version_added: "2.3"

**Comment by alxgu** [comment]
Please rebase your commits to one commit

**Review by alxgu — APPROVED** [review]

**Comment by tyouxa** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @tyouxa. To the current maintainers, @tyouxa please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with text 'shipit', 'needs_revision' or 'close_me' as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
Thanks @tyouxa for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

For help on how to do this cleanly please see http://docs.ansible.com/ansible/community.html#contributing-code-features-or-bugfixes

[This message brought to you by your friendly Ansibull-bot.]

**Pull request closed without merging**